### ISSUE TITLE
Standardize data fetching/preprocessing across segmentation and classification tasks

### ISSUE THREAD

**Issue by bdzyubak** [bdzyubak/Deep-Learning-Sandbox#15]
To implement the data fetching module (Source/shared_utils//prep_training_data.py), the segmentation and classification children classes were put together based on open source code. As a result, there are differences in how data is fetched, images are preprocessed, and train/test data is split. The intent is to have these the same by default, and have separate methods where needed. Update the code accordingly.

**Comment by bdzyubak** [comment]
One valid different that will not be standardized is the train/test split. Segmentation uses a random split. Classification uses sklearn.model_selection.train_test_split which makes training and validation sets balanced with respect to labels.
